src/interface/src/app/plan/create-scenarios/create-scenarios.component.scss

```diff
@@ -0,0 +1,42 @@
+.create-scenarios-panel {
+  background-color: white;
+  box-sizing: border-box;
+  height: 100%;
+  left: 0px;
+  padding: 32px 56px 32px 32px;
+  position: absolute;
+  width: 700px;
+  z-index: 2;
+}
+
+.create-scenarios-panel-expand-button {
+  align-items: center;
+  background-color: white;
+  border-radius: 8px;
+  display: flex;
+  height: 64px;
+  position: absolute;
+  right: -24px;
+  top: 12px;
+}
+
+.create-scenarios-panel-expand-button.collapsed {
```

Comment:
nit: .collapsed css can be nested inside .create-scenarios-panel-expand-button:

.create-scenarios-panel-expand-button {
  ...
  .collapsed {
    ...
}

Reply:
Oh I didn't know this, thanks!